stats/view/worker.go

```diff
@@ -104,25 +84,46 @@ func (v *View) Subscribe() error {
 	return <-req.err
 }
 
+// Subscribe begins collecting data for the given views.
+// Once a view is subscribed, it reports data to the registered exporters.
+func Subscribe(views ...*View) error {
+	var errstr []string
+	for _, v := range views {
+		err := v.Subscribe()
+		if err != nil {
+			errstr = append(errstr, fmt.Sprintf("%s: %v", v.Name, err))
+		}
+	}
+	if len(errstr) > 0 {
+		return errors.New(strings.Join(errstr, "\n"))
+	}
+	return nil
+}
+
 // Unsubscribe unsubscribes a previously subscribed view.
 // Data will not be exported from this view once unsubscription happens.
 func (v *View) Unsubscribe() error {
```

Comment:
This is now not reflexive, we have a struct with  a deprecated Subscribe method yet an Unsubscribe method, and a global Subscribe function, this is also inconsistent. Let's keep *view.View.Subscribe and still advertise Subscribe as a convenience function for subscribing many views.

Reply:
ok, although I think Subscribe as a method (as well as Unsubscribe) is weird because it's really an operation on the package global state rather than on the view on which the method is called

Reply:
What I'm going to do is convert Unsubscribe to a function as well and deprecate the existing method.